pyrelational/datasets/classification/mnist.py

```diff
@@ -0,0 +1,47 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor
+from torch.utils.data import ConcatDataset
+from torchvision import datasets, transforms
+
+from pyrelational.datasets.base import BaseDataset
+
+from .utils import create_splits
+
+
+class FashionMNIST(BaseDataset):
```

Comment:
Should the filename be changed to `fashion_mnist.py`?

Reply:
Yup that may be changed in the PR that this is currently adding to.